packages/inventory-next/src/utils/query.ts

```diff
@@ -0,0 +1,93 @@
+import { Brackets, EntityManager, FindOptionsWhere } from "typeorm"
```

Comment:
comment: A lot of stuff commented out here. Please add todos or remove.

Also, why are we mixing typeorm and MikroORM types? This module should not have a dep on typeorm